example/macos/example-macOS/example.entitlements

```diff
@@ -2,7 +2,11 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
-	<key>IDEDidComputeMac32BitWarning</key>
+	<key>com.apple.security.app-sandbox</key>
```

Comment:
Is this really required?

Reply:
for the example app....perhaps? I'm not an expert here but examples / dev builds usually go against apple's various sandbox systems

Reply:
I wrote this comment more for myself, just to check is it required, should i leave it or remove it. It is leftover from previous work. 😁